pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -389,11 +389,10 @@ func (oe *operationExecutor) DetachVolume(
 }
 
 func (oe *operationExecutor) VerifyVolumesAreAttached(
-	attachedVolumes []AttachedVolume,
-	nodeName types.NodeName,
+	attachedVolumesByNode map[types.NodeName][]AttachedVolume,
 	actualStateOfWorld ActualStateOfWorldAttacherUpdater) error {
 	volumesAreAttachedFunc, err :=
-		oe.operationGenerator.GenerateVolumesAreAttachedFunc(attachedVolumes, nodeName, actualStateOfWorld)
+		oe.operationGenerator.GenerateVolumesAreAttachedFunc(attachedVolumesByNode, actualStateOfWorld)
 	if err != nil {
```

Comment:
Ah - I see what you are saying.  I don't understand this well enough, but I don't think this PR changes the behaviour here.  That is not to say that we should not fix it, but I suggest a separate PR.

@jingxu97  / @saad-ali is this a (potential) problem?  Should we fix it (and here or in a separate PR)?